.github/workflows/reusable-web-e2e.yml

```diff
@@ -104,8 +108,7 @@ jobs:
       - run: |
           echo "BROWSER_PATH=$(which chrome)" >> $GITHUB_ENV
 
-      - name: Cypress run EE e2e
-        if: ${{ steps.setup.outputs.has_token == 'true' && inputs.ee }}
```

Comment:
Removed the Cypress EE E2E tests configuration, because of this the parallel tests execution wasn't working. We don't need to have it as a separate workflow step, as the `spec` prop allows passing multiple "targets", see the next comment.